slimmer.gemspec

```diff
@@ -17,12 +17,12 @@ Gem::Specification.new do |s|
   s.rubyforge_project = "slimmer"
 
   s.add_dependency 'nokogiri', '>= 1.5.0', '< 1.7.0'
-  s.add_dependency 'rack', '>= 1.3.5'
+  s.add_dependency 'rack', '~> 1.0'
   s.add_dependency 'plek', '>= 1.1.0'
   s.add_dependency 'json'
   s.add_dependency 'null_logger'
   s.add_dependency 'rest-client'
-  s.add_dependency 'activesupport'
+  s.add_dependency 'activesupport', '~> 4.0'
```

Comment:
Hmm, I think the reason I pinned it was because the newer version requires Ruby 2.2